compiler/ir/ir.tree/src/org/jetbrains/kotlin/ir/util/PatchDeclarationParents.kt

```diff
@@ -54,9 +54,15 @@ class PatchDeclarationParentsVisitor() : IrElementVisitorVoid {
     }
 
     override fun visitProperty(declaration: IrProperty) {
-        declaration.getter?.let { it.correspondingProperty = declaration }
-        declaration.setter?.let { it.correspondingProperty = declaration }
-        declaration.backingField?.let { it.correspondingProperty = declaration }
+        declaration.getter?.let {
```

Comment:
Please keep correspondingProperty  initialization

Reply:
I changed it because it's currently deprecated, but I can remove it from this PR.

Reply:
No, please support both correspondingPropertySymbol and correspondingProperty, otherwise we can break smth in Native

Reply:
Alright, I got rid of this part.

Reply:
@sfs 
Oh, you made right change, but I would prefer to also keep old assignment:
```
declaration.getter?.let { 
    it.correspondingProperty = declaration 
   it.correspondingProperty = declaration.symbol 
}
...
```

Reply:
@max-kammerer Done

Reply:
@sfs Sorry, but I made a big mistake: you original solution was totally right, I've rechecked implementation of corrrespondingProperty and found that it's delegates to correspondingPropertySymbol. So could you revert this change to your original solution

Reply:
@max-kammerer Done!